documentation (#6)
nat/ghtop: README.md

See what's happening on GitHub in real time (also helpful if you need to use up your API quota as quickly as possible)

With :heart: by @nat 

> ### `Installation Guide`
> 
> - It's better to have the latest version of python. 
> - Install following two python packages if you don't have them via pip. `emoji` and `enlighten`
> - Create `ghtoken.txt` and put your github access token in it. Follow [these](https://docs.github.com/en/free-pro-team@latest/github/authenticating-to-github/creating-a-personal-access-token) steps in order to find your access token.
> - Finaly, in order to see in realtime what's happening, run the following command from your terminal. `python ghtop.py tail` Feel free to pick one of the following options `quad`, `users`, `simple`, `tail`. I picked `tail` for this specific example.

Reviewer: ```suggestion
### Getting Started

- Install the `emoji` and `enlighten` packages with pip
- [Create a GitHub PAT](https://docs.github.com/en/free-pro-team@latest/github/authenticating-to-github/creating-a-personal-access-token) and put it in `ghtoken.txt`
```
